ok

```diff
@@ -0,0 +1 @@
+
```

Comment:
<?xml version="1.0" encoding="utf-8"?>
<resources>

<string name="app_name">Tip Calculator</string>
<string name="amount">Amount:</string>
<string name="fifteen_percent">15%</string>
<string name="eighteen_percent">18%</string>
<string name="custom_tip_percentage">Custom %</string>
<string name="tip">Total Tip</string>
<string name="total">Total per person</string>
<string name="partySize">number in party</string>
<string name="number">Number in party:</string>
    <string name="action_settings">action</string>

</resources>